src/ImageSharp.Web/Processors/ResizeWebProcessor.cs

```diff
@@ -134,7 +157,8 @@ public class ResizeWebProcessor : IImageWebProcessor
             Mode = GetMode(commands, parser, culture),
             Compand = GetCompandMode(commands, parser, culture),
             Sampler = GetSampler(commands),
-            PadColor = parser.ParseValue<Color>(commands.GetValueOrDefault(Color), culture)
+            PadColor = parser.ParseValue<Color>(commands.GetValueOrDefault(Color), culture),
+            TargetRectangle = targetRectangle
```

Comment:
I think we can just use `new(0 ,0, size.Width, size.Height)`. We don't do source/target rectangles in this library.

Reply:
Ah no... 

I meant don't add these sommnds at all and use the already parsed width/height.

Reply:
But then the rectangle will always be created when a width and height is set. Would that be ok. When we add them we are probably more flexible, or doesn't it make sense to use different once?